makejdk-any-platform.1

```diff
@@ -24,36 +24,36 @@ that you are building for further details.
 
 "./makejdk-any-platform.sh --docker jdk8u"
 
-This will start a Docker container and build you the latest Java 8 AdoptOpenJDK
-binary from the source at https://github.com/AdoptOpenJDK/openjdk-jdk8u
+This will start a Docker container and build you the latest Java 8 Temurin
+binary from the source at https://github.com/adoptium/openjdk-jdk8u
 
-Please visit https://www.adoptopenjdk.net for further support
+Please visit https://www.adoptium.net for further support
 
 .SH VERSIONS
 .TP
 .BR jdk8u
-Build Java 8, defaults to https://github.com/AdoptOpenJDK/openjdk-jdk8u
+Build Java 8, defaults to https://github.com/adoptium/openjdk-jdk8u
 .TP
 .BR jdk9u
-Build Java 9, defaults to https://github.com/AdoptOpenJDK/openjdk-jdk9u
+Build Java 9, defaults to https://github.com/adoptium/openjdk-jdk9u
```

Comment:
Looks like this is out of date versions wise and do we have all of these mirrored?